integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -168,8 +172,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.hostIP
-          - name: ALLUXIO_JOB_WORKER_JAVA_OPTS
-            value: {{ print " -Dalluxio.worker.hostname=$(ALLUXIO_WORKER_HOSTNAME) " | quote }}
```

Comment:
Same. Please bring these 2 lines back.

Reply:
I think it should use  the ALLUXIO_JOB_WORKER_JAVA_OPTS in configmap. Otherwise it will lose a lot of other configurations. I've tested it in our environment.

https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml#L41